tech-jobs/summary_stats/scripts/main.py

```diff
@@ -10,77 +10,65 @@
 from entryProcessor import EntryProcessor
 from employee import Employee
 from records import Records
+from csv_parser  import csv_parser
+from json_parser import json_parser
 
-empl_path = sys.argv[1] #directory of csv files to process
-primary_skills = sys.argv[2].split(',') # filter by primary skill, "all" for alll skills, -(skill) to exclude
-target = sys.argv[3] #csv file name to write
-
-exclusive = False
-if primary_skills[0][0] == '-':
-    exclusive = True
-    all_skills_but = []
-    for skill in primary_skills:
-        all_skills_but.append(re.sub(r'[-()]','', skill))
-    all_skills_but.append('-1')
+try:
+    assert len(sys.argv) == 5
```

Comment:
There is a very nice library called argparse which does all of this for you in a much more elegant way--consider switching to that rather than raising exceptions based on argv length--you don't know whether you have been passed the right types in the right order... Or which one is missing...